stable/dynamic-gateway-service/templates/_apiconnectGateway.tpl

```diff
@@ -29,16 +29,19 @@ auto-startup.cfg: |
       ssl-config-type server
     exit
 
-{{- if .Values.datapower.restManagementState }}
-{{- if eq .Values.datapower.restManagementState "enabled" }}
+{{- if or (eq (.Values.datapower.restManagementState | default "disabled") "enabled") (eq (.Values.datapower.apiDebugProbe | default "disabled") "enabled") }}
+{{  if eq (.Values.datapower.restManagementState | default "disabled") "enabled" }}
+    # REST Management explicitly enabled
+{{  else }}
+    # REST Management implicitly enabled because an API Debug Probe is enabled
```

Comment:
What is the purpose of this if-else block? It seems to do nothing, as it ends before any actual config is templated.

Reply:
It drops a comment into `auto-startup.cfg` describing specifically why RMI was enabled: 
```shell
$ kubectl exec -it r47313e0ff9-dynamic-gateway-service-0 -- \
          cat /drouter/config/auto-startup.cfg | grep -A6 '# REST'
# REST Management implicitly enabled because an API Debug Probe is enabled
rest-mgmt
  admin-state enabled
  local-address 0.0.0.0
  port 5554
  ssl-config-type server
exit

```
It may not be totally necessary, but since these changes allow RMI to be implicitly enabled, I figure it might be useful to drop an explanation into the generated config.